blog.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial=scale=1">
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+    <title>Blog</title>
```

Comment:
One thing that'll is standard -ish is using a `|` character as a divider to show the site name and the sub page. That way when looking at tabs it's nice to see the site name, rather than "Blog"

`<title>Pizza Recipe | Home</title> `